cmd/minikube/cmd/config/config_test.go

```diff
@@ -41,12 +41,14 @@ var configTestCases = []configTestCase{
     "ReminderWaitPeriodInHours": 99,
     "cpus": 4,
     "disk-size": "20g",
+    "iso-url": "http://foo.bar/minishift-centos.iso",
     "log_dir": "/etc/hosts",
     "show-libmachine-logs": true,
     "v": 5,
     "vm-driver": "kvm"
 }`,
 		config: map[string]interface{}{
+			"iso-url": "http://foo.bar/minishift-centos.iso",
```

Comment:
I think we can add a comment on top of that file and warn user not to edit it manually and use config set command.